ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml

```diff
@@ -644,7 +644,7 @@ StatusSectionLayout {
                 return
             }
 
-            Global.displayToastMessage(title, qsTr("View on etherscan"), "",
+            Global.displayToastMessage(title, url === "" ? qsTr("Something went wrong") : qsTr("View on etherscan"), "",
```

Comment:
I'd be more descriptive here, like "Error deploying assets"? (not sure about the actual message)

Reply:
Originally I wanted to expose the underlying error message that we get when the transaction fails, but unfortunately we can't easily access it with our current backend implementation so I went with something generic.

I can update this to something you've mentioned, chances are low that there'd be any other semantic error when `url == ""`

Reply:
Yeah, not a big deal